requirements.txt

```diff
@@ -0,0 +1,3 @@
+# generated from manifests external_dependencies
+pandas
```

Comment:
Why do we need the pandas library? It's a very heavy library.
---> https://docs.python.org/3/library/csv.html

Reply:
It's a very heavy library, as I mentioned before. I've given you an alternative for CSV; Odoo already uses libraries for this; check which ones it uses.